example-workflows/review-app.yaml

```diff
@@ -23,6 +23,10 @@ jobs:
           # create a review app
           command: review-apps:create
           git_remote_url: 'ssh://dokku@dokku.me:22/appname'
+          # specify `--force` as a flag for git pushes
+          git_push_flags: '--force'
+          # specify the name of the branch, in cas you do not use master
+          # branch: main
```

Comment:
I think this should be removed tbh. There are other examples showing this. The review-app example isn't meant to be a kitchen sink, just what would make the 90% case work. If you normally deploy a different branch, then that should be configured in the app configuration (and therefore cloned). If it isn't cloned, then thats a bug in Dokku.